inc/assets/js/parts/_main_sticky_footer.part.js

```diff
@@ -0,0 +1,101 @@
+var czrapp = czrapp || {};
+/************************************************
+* STICKY FOOTER SUB CLASS
+*************************************************/
+(function($, czrapp) {
+  var _methods =  {
+    init : function() {
+      this.$_push   = $('#tc-push-footer');
+      this._class   = 'sticky-footer-enabled';
+      this.$_page   = $('#tc-page-wrap');
+      
+      if ( 1 != TCParams.stickyHeader ) {//sticky header fires a resize
+        var self = this;
+        setTimeout( function() {
+                self._apply_sticky_footer(); }, 50 
+        );
+      }
+    },
+
+    /***********************************************
+    * DOM EVENT LISTENERS AND HANDLERS
+    ***********************************************/
+    stickyFooterEventListener : function() {
+      var self = this;
+
+      // maybe apply sticky footer on window resize
+      czrapp.$_window.on( 'tc-resize', function() {
```

Comment:
this should be $_body too, but anyway it propagates so.. no problem
or maybe it should be fired on $_window in czrapp since it regards to the $(window)